common/features/config/networks/static/reducer.ts

```diff
@@ -851,7 +852,33 @@ export const STATIC_NETWORKS_INITIAL_STATE: types.ConfigStaticNetworksState = {
       initial: 1
     }
   }
-};
+},
+
+  ETI: {
+    id: 'ETI',
+    name: 'Etherinc',
+    unit: 'ETI',
+    chainId: 101,
```

Comment:
I recommend waiting until after WebChain's hardfork before proceeding with this.  It's great that somebody has changed the chain ID, but until the change takes affect, applying this now would be dangerous for users.

Reply:
Also, *very* important:  Please confirm that EtherInc supports `EIP-155`.  This is a requirement for inclusion in MyCrypto.

Reply:
> Also, _very_ important: Please confirm that EtherInc supports `EIP-155`. This is a requirement for inclusion in MyCrypto.

Yes, EtherInc supports EIP-155

Reply:
> I recommend waiting until after WebChain's hardfork before proceeding with this. It's great that somebody has changed the chain ID, but until the change takes affect, applying this now would be dangerous for users.

Ok will wait for webchain fork